hummingbird/ml/operator_converters/_label_encoder_implementations.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, logical_operator, classes, device, extra_config={}):
 
         # Sort the classes and convert to torch.int32
         self.max_word_length = self.max_word_length // 4
-        classes_conv = torch.from_numpy(np.array(sorted(set(classes)), dtype=data_type).view(np.int32))
+        classes_conv = torch.from_numpy(np.array(sorted(set(classes)), dtype=data_type).view(np.int32)).detach().clone()
         classes_conv = classes_conv.view(1, -1, self.max_word_length)
 
         self.condition_tensors = torch.nn.Parameter(torch.IntTensor(classes_conv), requires_grad=False)
```

Comment:
Since we are at it, can you also please remove `torch.IntTensor`. It is not necessary here anymore.